cypress/e2e/plugins/security-dashboards-plugin/oidc_test.cy.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+describe('Log in via OIDC', () => {
+  const login = 'admin';
+  const password = 'admin';
+
+  const kcLogin = () => {
+    cy.get('#kc-page-title').should('be.visible');
+    cy.get('#username').type(login);
+    cy.get('#password').type(password);
+    cy.get('#kc-login').click();
+  };
+
+  const logout = () => {
+    cy.get('#user-icon-btn').should('be.visible', { timeout: 15000 });
+    cy.get('#user-icon-btn').click();
+    cy.get('button[data-test-subj^="log-out-"]').click();
+    cy.get('#kc-page-title').should('be.visible');
+  };
+
+  afterEach(async () => {
+    logout();
+  });
+
+  it('Login to app/opensearch_dashboards_overview#/ when OIDC is enabled', () => {
```

Comment:
This looks great. Its concise and clear what is being tested. Nice! I like the shorthand `kcLogin` for login to keycloak.

Reply:
@jakubp-eliatra 🙌